packages/vuepress-theme-odyssey/package.json

```diff
@@ -9,6 +9,7 @@
   "devDependencies": {
     "@okta/odyssey": "^0.6.0",
     "@okta/odyssey-icons": "^0.6.0",
+    "eslint-config-vuepress": "^2.2.0",
```

Comment:
Since this is likely going to be used on the docs site as well, I'd suggest moving this to the root dependency list.

Reply:
Done